### PR TITLE
Notifications to active users only

### PR DESCRIPTION
**Pull request by bwilliams3**
Addresses issue #7506

### CODE REVIEW

File: wagtail/admin/tests/pages/test_edit_page.py [hunk 1]
```diff
@@ -2564,3 +2564,40 @@ def test_updated_comments_notifications_profile_setting(self):
...
+
+        self.assertRedirects(response, reverse('wagtailadmin_pages:edit', args=[self.child_page.id]))
+
+        # Check the comment was added
+        comment = self.child_page.comments.get()
+        self.assertEqual(comment.text, 'A test comment')
+
+        # No emails should be submitted because subscriber is inactive
+        self.assertEqual(len(mail.outbox), 0)
+
```
Comment (lb-) [inline wagtail/admin/tests/pages/test_edit_page.py:2603]: I think the linting is flagging this extra line here as an issue.

**Comment by lb-** [comment]
@bwilliams3  thanks so much for taking the time to work through this problem, find the solution and also include unit tests.

There appears to be a small linting issue, you can see these issues if you run `make lint` in your terminal with the wagtail project open. See more info here https://docs.wagtail.io/en/stable/contributing/python_guidelines.html#pep8

```
./wagtail/admin/tests/pages/test_edit_page.py:2603:1: W391 blank line at end of file
```

**Review by lb- — CHANGES_REQUESTED** [review]
Just one minor linting issue to fix.

**Review by lb- — APPROVED** [review]

**Comment by lb-** [comment]
@bwilliams3 I just fixed up that minor linting issue, validated this behaviour locally (before & after commits) and it works great.

I have pushed to your branch and this is now merged in, please check the contributors listing and let me know if you want your name listed differently.

Thanks again for taking the time to provide this fix, it is really appreciated!

**Comment by bwilliams3** [comment]
@lb- No problem and thanks for wrapping up that last fix for me -- appreciate it!

**Pull request merged**